Jenkinsfile

```diff
@@ -354,7 +354,7 @@ pipeline {
                     ./unzipped/cert-generator.sh -d "${domain()}" -n ${jobBaseName()}-${BUILD_NUMBER}-scf -o unzipped/certs
 
                     suffix=""
-                    if echo "$FISSILE_STEMCELL" | grep -qv "fissile-stemcell-sle"; then
+                    if echo "${params.FISSILE_STEMCELL}" | grep -qv "fissile-stemcell-sle"; then
```

Comment:
I think the reason we `source` the .envrc above is to make `FISSILE_STEMCELL` available in `sh`. Now we get the stemcell in Jenkins space so is there still a reason to source .envrc?